Externals/expr/include/expr.h

```diff
@@ -468,6 +479,17 @@ extern "C" {
       }
       return i;
     }
+    else if (c == '"') {
+      i++;
+      while (i < len && s[i] != '"')
+        i++;
+
+      if (s[i] != '"' || i <= 1)
```

Comment:
All this check needs to do is detect if we hit the end of the string, i.e. `i >= len`